data/json/items/gun/223.json

```diff
@@ -404,21 +465,82 @@
+    "type": "GUN",
+    "name": { "str": "M16 Auto Rifle" },
+    "description": "The M16 is a very common assault rifle descended from the AR-15, used by militaries across the world for over 50 years.  Unlike its more common brother, the M16 Burst Rifle, this one has full-auto capabilities.",
+    "variants": [
+      {
+        "id": "m16a3",
+        "name": { "str": "M16A3" },
+        "description": "The M16 is a very common assault rifle descended from the AR-15, used by militaries across the world for over 50 years.  It is a gas operated, rotating bolt rifle known for its accuracy and controllable recoil.  This one is a relatively rare M16A3, with full auto capability.",
+        "weight": 1
+      },
+      {
+        "id": "acr",
+        "name": { "str": "Remington ACR" },
+        "description": "This is a shorter M4 carbine with a 10.3 inch barrel, intended for confined spaces and close quarters battle situations."
+      }
+    ],
+    "modes": [ [ "DEFAULT", "semi-auto", 1 ], [ "AUTO", "auto", 4 ] ]
+  },
+  {
+    "//": "Decided to give this one a non-brand name, as it is related to the M16. It was made with an open-bolt design, so I figured this name was reasonable.",
     "id": "m231pfw",
-    "copy-from": "m16a4",
+    "copy-from": "m16_burst_rifle",
     "type": "GUN",
-    "name": { "str": "M231 Port Firing Weapon" },
-    "description": "The M231 PFW is an adaptation of the M16 assault rifle, designed to be used in the firing ports of the M2 Bradley Infantry Fighting Vehicle.  It does not have a buttstock or front sight post, and the smooth front hand guard makes it slighly less ergonomic.  It fires from an open bolt.  The fire selector says only 'SAFE' and 'AUTO'.  You can't help but wonder which one is preferable.",
+    "name": { "str": "M16 Open-Bolt Rifle" },
+    "description": "The M16 Open-Bolt Rifle is an M16 rifle modified to fire with an open-bolt design. This makes the rifle fire at much higher speeds than most other assault rifles, but at the cost of accuracy.",
     "weight": "3330 g",
     "volume": "2400 ml",
     "longest_side": "727 mm",
     "price": 90000,
     "price_postapoc": 3500,
+    "variants": [
+      {
+        "id": "m231pfw",
+        "name": { "str": "M231 Port Firing Weapon" },
+        "description": "The M231 PFW is an adaptation of the M16 assault rifle, designed to be used in the firing ports of the M2 Bradley Infantry Fighting Vehicle.  It does not have a buttstock or front sight post, and the smooth front hand guard makes it slighly less ergonomic.  It fires from an open bolt.  The fire selector says only 'SAFE' and 'AUTO'.  You can't help but wonder which one is preferable."
+      }
+    ],
     "dispersion": 350,
-    "//": "This weapon has an extremely high rate of fire, due to the open bolt design. This is based on a cyclic rate of fire of 1150 RPM, which is close to 19 rounds per second. 6 rounds is comparable to to other weapons with similar rates, such as the MAC-11.",
```

Comment:
Please keep this comment, it is still useful.

Reply:
got it, I'll add it back in and remove the old comment